modules/core/arrow-typeclasses/src/main/kotlin/arrow/typeclasses/Composed.kt

```diff
@@ -211,20 +348,20 @@ interface ComposedBifoldable<F, G> : Bifoldable<Nested<F, G>> {
 
   fun G(): Bifoldable<G>
 
-  override fun <A, B, C> BiunnestedType<F, G, A, B>.bifoldLeft(c: C, f: (C, A) -> C, g: (C, B) -> C): C = F().run {
+  override fun <A, B, C> BinestedType<F, G, A, B>.bifoldLeft(c: C, f: (C, A) -> C, g: (C, B) -> C): C = F().run {
```

Comment:
Are we sure of this change, or was it part of the merge?

Reply:
I've checked the history and this is part from our changes, related to https://github.com/arrow-kt/arrow/pull/975/commits/fa50cf1debda58318b1ae59d3666a10d1aeca685

Reply:
Right, right, they're correct, there's a couple of fixes to other that were wrong.

Cool, only the test left then.